index.html

```diff
@@ -10,17 +10,19 @@
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Press+Start+2P&display=swap" rel="stylesheet">
     <script src="./script.js" defer></script> 
-    <title>Surfing Penguins - JS Game. </title>
+    <title>SURFWAVE Fury</title>
 </head>
 <body>
     <main>
         <div class="landingTopContainer">
-            <div class="landingLeft"></div>
+            <div class="stopwatchContainer">
+                <p class="stopwatchText">Time: <span id="stopwatchSecondsDisplay">0</span>s</p>
+            </div>
             <div class="landingTitleContainer">
-                <h1 class="landingTitle">SURFWAVE Fury</h1>
+                <h1 class="landingTitle">Surfwave Fury</h1>
             </div>
             <div class="leaderSurfboardContainer">
-                <!-- <button id="leaderSurfboard">Leader Surf Board</button> -->
+                <button id="leaderSurfboard">Leader Surf Board</button>
```

Comment:
button needs to not be there because it is from a future story pls

Reply:
This has been removed.